src/main.rs

```diff
@@ -134,6 +134,13 @@ pub async fn main() -> Result<(), anyhow::Error> {
             .takes_value(true)
             .multiple(true)
         )
+        .arg(Arg::with_name("env_files")
```

Comment:
Nit: Should be "env_file" without the "s"?

Reply:
I thought that if `multiple(true)`, then we made the name plural to indicate that one needed to load them all.

(Why is this `multiple(true)`? So that we can do `--env-file global.env --env-file stage.env`)

Reply:
I just updated the help string for that.

Reply:
Ah ok. I was mostly looking at the examples and help text if I am being honest. Missed the multiple part